db/migrate/20170530221538_remove_colors_from_pieces.rb

```diff
@@ -0,0 +1,7 @@
+class RemoveColorsFromPieces < ActiveRecord::Migration[5.0]
+  def change
+  	remove_column :pieces, :black, :boolean
+  	remove_column :pieces, :white, :boolean
+  	add_column :pieces, :black?, :boolean
```

Comment:
This should probably be named something like `is_black` since this is a database column name - apparently it accepts a `?` in the name, but that may cause issues with ActiveRecord (since AR creates methods named like`COLUMN_NAME?` for ever column - not sure what it will do with `black??`)